dspace-spring-rest/src/test/java/org/dspace/app/rest/MetadatafieldRestRepositoryIT.java

```diff
@@ -124,4 +163,190 @@ public void findByNullSchema() throws Exception {
                    .andExpect(status().isUnprocessableEntity());
     }
 
+    @Test
+    public void createSuccess() throws Exception {
+
+        MetadataFieldRest metadataFieldRest = new MetadataFieldRest();
+        metadataFieldRest.setElement(ELEMENT);
+        metadataFieldRest.setQualifier(QUALIFIER);
+        metadataFieldRest.setScopeNote(SCOPE_NOTE);
+
+        String authToken = getAuthToken(admin.getEmail(), password);
+        AtomicReference<Integer> idRef = new AtomicReference<>();
+
+        try {
+            assertThat(metadataFieldService.findByElement(context, metadataSchema, ELEMENT, QUALIFIER), nullValue());
+
+            getClient(authToken)
+                    .perform(post("/api/core/metadatafields")
+                            .param("schemaId", metadataSchema.getID() + "")
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isCreated())
+                    .andDo(result -> idRef.set(read(result.getResponse().getContentAsString(), "$.id")));
+
+            MetadataField metadataField = metadataFieldService.find(context, idRef.get());
+            assertThat(metadataField, notNullValue());
+
+            assertEquals(metadataField.getMetadataSchema(), metadataSchema);
+            assertEquals(metadataField.getElement(), ELEMENT);
+            assertEquals(metadataField.getQualifier(), QUALIFIER);
+            assertEquals(metadataField.getScopeNote(), SCOPE_NOTE);
+
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
+    @Test
+    public void createUnauthauthorized() throws Exception {
+
+        MetadataFieldRest metadataFieldRest = new MetadataFieldRest();
+        metadataFieldRest.setElement(ELEMENT);
+        metadataFieldRest.setQualifier(QUALIFIER);
+        metadataFieldRest.setScopeNote(SCOPE_NOTE);
+
+        try {
+            getClient()
+                    .perform(post("/api/core/metadatafields")
+                            .param("schemaId", metadataSchema.getID() + "")
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isUnauthorized());
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
```

Comment:
add a test for the forbidden scenario (loggedin user without  admin rights)

Reply:
This test was added.  See `createUnauthorizedEPersonNoAdminRights()` in the code.